src/components/Drawer/Drawer.Title.tsx

```diff
@@ -17,6 +17,7 @@
  */
 
 import { ReactElement, useCallback, useMemo } from 'react'
+import { PiBookOpen } from 'react-icons/pi'
```

Comment:
wouldn't it be better for this icon to be taken from our package rather than react-icons? 🤔

Reply:
Internally I preferred using icons directly from `react-icons` to avoid coupling between the build of our icons package and the components

Having said that, in terms of bundle size is the same so we can go both ways

Reply:
>to avoid coupling between the build of our icons package and the components

It makes sense for DevX and contribution, however this could make it possible to validate the icon builder script anytime during development since you are required to use it post-install. Does it make sense to you?

Reply:
we decided to open an issue on this